test/helper/Puppeteer_test.js

```diff
@@ -22,7 +22,7 @@ describe('Puppeteer - BasicAuth', () => {
     global.codecept_dir = path.join(__dirname, '/../data');
 
     I = new Puppeteer({
-      url: 'http://localhost:8000',
+      url: siteUrl,
```

Comment:
small fix for running tests (locally at least).  I found that when running `docker-compose run --rm test-helpers test/helper/Puppeteer_test.js` that using `localhost` would error out with `Error: net::ERR_CONNECTION_REFUSED at http://localhost:8000/basic_auth`